processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -117,22 +117,25 @@ public static void createLocations(String[] locations) {
       }
     }
   }
+
   /**
+   *
    * This method will form the local data folder store location
    *
-   * @param databaseName
-   * @param tableName
+   * @param carbonTable
    * @param taskId
    * @param partitionId
    * @param segmentId
+   * @param isCompactionFlow
+   * @param isAltPartitionFlow
    * @return
    */
-  public static String[] getLocalDataFolderLocation(String databaseName, String tableName,
+  public static String[] getLocalDataFolderLocation(CarbonTable carbonTable,
```

Comment:
Dont remove parameter tableName, you can remove databaseName, as in case of Pre aggregate table, table name can be of child even though carbonTable object is for main table

Reply:
done